Raven.Database/Storage/Esent/SchemaCreator.cs

```diff
@@ -1131,7 +1131,7 @@ internal void CreateIndexedDocumentsReferencesTable(JET_DBID dbid)
                         cbKeyMost = SystemParameters.KeyMost,
                         cbVarSegMac = SystemParameters.KeyMost,
                         szKey = by_view_and_key,
-                        grbit = CreateIndexGrbit.IndexDisallowNull | CreateIndexGrbit.IndexUnique,
+                        grbit = CreateIndexGrbit.None,
```

Comment:
In schema 5.4 we had `CreateIndexGrbit.IndexDisallowNull` here - we should have the same now